libraries/AP_Motors/examples/AP_Motors_test/AP_Motors_test.cpp

```diff
@@ -111,7 +111,7 @@ void motor_order_test()
     hal.console->printf("testing motor order\n");
     motors.armed(true);
     for (int8_t i=1; i <= AP_MOTORS_MAX_NUM_MOTORS; i++) {
```

Comment:
Why is there no white space in this state? I think that it is better to combine with others.